crates/e2store/README.md

```diff
@@ -15,20 +15,20 @@ era is a format for storing beacon chain data more information can be found here
 
 era1 is a format for storing all of Ethereum's pre merge blocks. It contains block headers, block bodies, and receipts for pre-merge block history which ranges block 0-15537394
 
-## What is era2?
+## What is e2ss?
 
-era2 is a format made to store full flat state snapshots, one of our first uses of this will be using to bootstrap Portal State Network bridges. Unlike `.era`/`.era1` era2 files will only store 1 block's worth of state per file. The reason for this choice is a snapshot of the state is quite large.
+e2ss is a format made to store full flat state snapshots, one of our first uses of this will be using to bootstrap Portal State Network bridges. Unlike `.era`/`.era1` e2ss files will only store 1 block's worth of state per file. The reason for this choice is a snapshot of the state is quite large.
 
 TODO: Add chart of snapshot size at every million block interval.
 
-### Era2 analysis tool
+### E2ss analysis tool
```

Comment:
And here we use `E2ss`... Though, I get it's the first word which is usually capitalized... but, I'd still rather see `e2ss` or `E2SS` here